portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/* Create a new HTML element */
+function createElementWithParams (tagName, id, classes, events, innerText) {
```

Comment:
We could use an object parameter with destructured default parameter

https://www.sitepoint.com/es6-default-parameters/

This link actually has an example that's super similar to what you are doing.

Reply:
Oh, good idea!